src/app/data-planes/data/index.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  DataPlaneOverview,
+} from '@/types/index.d'
+
+export function parseMTLSData(dataPlaneOverview: DataPlaneOverview) {
+  if (dataPlaneOverview.dataplaneInsight === undefined || dataPlaneOverview.dataplaneInsight.mTLS === undefined) {
+    return null
+  }
+
+  const rawExpDate = new Date(dataPlaneOverview.dataplaneInsight.mTLS.certificateExpirationTime)
+  // this prevents any weird date shifting
+  const fixedExpDate = new Date(rawExpDate.getTime() + rawExpDate.getTimezoneOffset() * 60000)
+
+  return {
+    ...dataPlaneOverview.dataplaneInsight.mTLS,
+    certificateExpirationTime: fixedExpDate.toISOString(),
```

Comment:
I thought there was a t funciton or something like this for date display no?

Reply:
Yeah that used in the template now as its a view/format thing, ideally we would be using `Date`s here instead of string also 🤔 let me look at that actually, I might move the `toISOString` to the view so we are dealing with correct data types in the data layer.

Reply:
Just took a quick look at this, and I had second thoughts, the rest of the dates coming from the API are all in a string format, so will leave them all the same.